ovs/lib/helpers/celery_toolbox.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2016 iNuron NV
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Module containing celery helpers
+"""
+
+import datetime
+import time
+from celery.task.control import revoke
+from ovs.dal.lists.storagerouterlist import StorageRouterList
+from ovs.extensions.generic.sshclient import SSHClient
+from ovs.extensions.services.service import ServiceManager
+from ovs.log.logHandler import LogHandler
+
+logger = LogHandler.get('lib', name='celery toolbox')
```

Comment:
Please define the ```logger``` instance inside the method using it
We're trying to minimize the variable declarations outside of ```main``` and/or classes